src/components/elements/Loading.tsx

```diff
@@ -8,6 +8,8 @@ const Loading = () => (
     height='100%'
     data-testid='loading'
     sx={{ p: 10 }}
+    aria-live='polite'
+    aria-busy='true'
```

Comment:
Interesting. I don't totally understand what these do but I created a ticket https://www.pivotaltracker.com/story/show/186561233 for assessing and improving accessibility of loading states